java/code/src/com/suse/manager/webui/templates/system-common.jade

```diff
@@ -14,6 +14,10 @@ else
 
 .spacewalk-toolbar-h1
   .spacewalk-toolbar
+    if !server.isConvertibleToProxy()
```

Comment:
Shouldn't this be the other way around? We want to show this link if the machine is convertible to proxy, right?